packages/core/src/internal/clients/pubsub/AbstractPubsubClient.ts

```diff
@@ -260,7 +260,7 @@ export abstract class AbstractPubsubClient<TGrpcError>
     // transparently restart the stream instead of propagating an error.
     if (shouldReconnectSubscription) {
       options.restartedDueToError = true;
-      const reconnectDelayMillis = 100;
+      const reconnectDelayMillis = 5000;
```

Comment:
I think let's be careful not to conflate 2 things here.

Thing 1: what is a good value to use here for UX. It might be higher than 100 but it's probably less than 5000.
Thing 2: regardless of connect/disconnect status, you will only see a discontinuity if you miss messages. The server buffers the last 5 seconds worth of messages, so if you reconnect in less than 5s, you won't get one. But I think that for your demo purposes if you leave the connection blocked for more than 5s, you will get a discontinuity even if this reconnect attempt is happening every 100ms?

Reply:
ah yeah you're right, I forgot about the 5s buffer 🙃

do you think it would still be worthwhile to increase the delay from 100 to maybe 1000? 
otherwise I can just close this pr

Reply:
i think we should start a thread about that in slack. i can seed it.